app/assets/javascripts/lat-long-info.js

```diff
@@ -0,0 +1,52 @@
+/* global FileReader EXIF $ */
+
+$(function () {
+  var sightingImageEl = document.getElementById('sighting_image')
```

Comment:
It seems like we're using jQuery here from the `$` on line `3` I recommend continuing to use jQuery for the rest of the file instead of using plain Javascript.